wordpress-mobile/AztecEditor-Android
# Issue/373 horizontal rule button

### Fix
Add horizontal rule format button to the format toolbar as described in https://github.com/wordpress-mobile/AztecEditor-Android/issues/373.

### Test
1. Notice ***Horizontal Rule*** format button.
2. Tap ***Horizontal Rule*** format button.
3. Notice horizontal rule is added at cursor location.

#### Note
The horizontal rule format button icon will appear slightly larger than the other icons due to 71c83fd93e2b0a42b59a0c6a0e1ffcc9a41b6e69.  The other icons are updated in https://github.com/wordpress-mobile/AztecEditor-Android/pull/387.
